src/sortStudents.ts

```diff
@@ -1,16 +1,56 @@
-
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(students:Student[], sortBy: SortType,
+  order: SortOrder): Student[] {
+  const res = [...students];
+
+  const getAverage = (array: number[]): number => array
+    .reduce((acc, el) => acc + el, 0) / array.length;
+
+  res.sort((firstEl, secondEl) => {
+    const orderElement = {
+      first: firstEl,
+      second: secondEl,
+    };
+
+    if (order === 'desc') {
+      orderElement.first = secondEl;
+      orderElement.second = firstEl;
+    }
```

Comment:
_comment_
you can use ternary operator here. But it is just a comment, can't see any real benefits exept less code.
```suggestion
    const orderElement = {
      first: order === 'asc' ? firstEl : secondEl,
      second: order === 'asc' ? secondEl: firstEl,
    };
```